docs/basics/index.md

```diff
@@ -1,5 +1,18 @@
 # Basic Usage
 
+## Getting Started
```

Comment:
Why did you need to move this? Was it required to fix some warnings or is it just a matter of preference?

I don't know if I think it's better now or not. In a way, I never liked the name "Basic usage", but I couldn't come up with anything else when trying to distinguish that from the more advanced features such as making custom interactive interfaces.

In other projects, this section is just called "User guide", but I thought the customized figures are also a guide for the user, so I didn't think it made so much sense to name it "User guide".

Maybe we could keep the "Getting started" section and inside add installation instructions? Right now they are just on the home page.

Reply:
> Why did you need to move this? Was it required to fix some warnings or is it just a matter of preference?

Because there would have been too many items in the navbar at the top. It is already quite long.
With the new theme we have to have only a small number of top-level sections, in my opinion.

The 'Basic usage' section (except for the 'getting started' pages I merged) looks like it could be moved into the gallery. It is a little different because it shows several customisation options per plot type. But I still think it makes sense to move it there.

> Maybe we could keep the "Getting started" section and inside add installation instructions? Right now they are just on the home page.

There definitely should be installation instructions in 'getting started' :+1:

Reply:
> Why would there be too many items? Did you add an item in the navigation? I thought it worked ok as it is?

The dev docs are now. This is what it looks right now:

![Screenshot_20231206_134748](https://github.com/scipp/plopp/assets/11393224/e306d259-394c-4a27-bc9b-d24fab59c47a)

Or should we merge basic usage and custom figures? Both are kind of part of a user guide.

Reply:
Ah it's because of the developer docs. "Developer documentation" takes quite a lot of space as a title.
Does it make sense to have the developer docs under the `More` menu? I know it's not like the other projects, but there is not as much info in there as in Scipp for example.

Reply:
So make a "User guide" page where you would have:
- "Getting started" (installation, overview, numpy pandas xarray)
- "Plot types" (to replace "By examples"). The "Plot types" name is from Matplotlib but maybe we can find a slightly better one?
- "Custom figures" (which could be displayed also as a grid of cards?)

Reply:
Done. Have a look at the CI-rendered docs.